docs/blogs/links-syphon/syphon.md

```diff
@@ -0,0 +1,151 @@
+# Mixing Metadata, Air and Water: Use the lakeFS Airflow Provider to Link Airflow Execution to lakeFS Data
+
+By [Ariel Shaqed (Scolnicov)]
+
+## Introduction
+
+"How do I integrate X with lakeFS" is an ever-green question on [lakeFS
+Slack][lakefs-slack].  lakeFS takes a "tooling-first" strategy to data
+management: it slots into your existing lineup of tools.  So a significant
+part of our work on lakeFS is devoted to leveraging lakeFS and these other
+tools to improve these integrations.  Our latest addition is Project Syphon!
+
+<img alt="Soda syphon, by Avi Nahmias, available from https://commons.wikimedia.org/wiki/User:Avin" src="https://upload.wikimedia.org/wikipedia/commons/1/1c/Soda_syphon_IMG_1505C.jpg" width=400/>
+
+Airflow was one of our earliest integrations: Itai published the original
+[Air and Water][airflow-air-and-water-blog-1] 2 years ago.  It's time to
+deepen our integrations!  The latest release of the lakeFS provider for
+Airflow, codename "Project Syphon", adds deep linking and metadata between
+Airflow DAG runs and the lakeFS commits that they create.  It connects the
+two for interactive or programmatic use.
+
+The new Airflow operators add formatted metadata to your commits.  lakeFS
+uses this metadata to link to Airflow.  Project Syphon adds flexibility to
+these keys by defining a format for metadata keys.  You can leverage such
+metadata to add your own operable metadata on any commits you perform from
+_any_ tools (not only from Airflow).
+
+How are you integrating these new features into your workflows?  How can we
+improve them and help you?  Please let us know on [lakeFS
+Slack][lakefs-slack] or by [opening an issue][airflow-provider-new-issue]!
+
+## Using the lakeFS Airflow provider
+
+Links are generated by lakeFS provider versions 0.46.1 and above.  To use
+them, ensure your Airflow `requirements.txt` includes
+
+```py
+airflow-provider-lakefs>=0.46.1
+```
+
+That is pretty much all.  Using LakeFSCommitOperator or LakeFSMergeOperator
+normally in your DAGs will add metadata to your commits.
+
+Any version of lakeFS can serve commit metadata to clients.  The lakeFS GUI
+adds the ability to interpret Airflow provider metadata to link back to
+Airflow starting at version **TBD TBD** 0.100.0 **TBD TBD**.
+
+### Metadata on lakeFS commits
+
+When examining a commit in lakeFS, the UI shows metadata and a link "Open
+Airflow UI" to the DAG run in the UI.
+
+![lakeFS commit UI with "Open Airflow UI" button][lakefs-commit-ui-open-button]
```

Comment:
Change the cropping here so that either (a) it includes the lakeFS header bar to make it clear where it comes from or (b) remove the `committing to lakeFS using airflow!` and top-right icons because it's not clear that they are part of the screenshot and not the article body text

Reply:
This is very true, but the lakeFS GUI is not particularly amenable to doing this: adding the header bar makes the commit metadata box really small.  So I did something else: frame the pictures.